bin/docbookrx

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env ruby
 
-if File.exist?(app_lib_path = File.join(File.dirname(__FILE__), '../lib/docbookrx'))
+
+if File.exist?(app_lib_path = File.expand_path(File.join(File.dirname(__FILE__), '../lib/docbookrx')))
```

Comment:
I tried the same thing for asciidoctor, and I get the same exception:

kjell@pc asciidoctor]$ bin/asciidoctor
/usr/share/rubygems/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- bin/../lib/asciidoctor (LoadError)
	from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:54:in `require'
	from bin/asciidoctor:6:in `<main>'
[kjell@pc asciidoctor]$ 

This is not important tom me that you include this change, so just leave it out if you like.